extensions/amp-analytics/0.1/vendors/permutive.json

```diff
@@ -1,9 +1,10 @@
 {
   "vars": {
-    "identity": "${clientId(_ga)}"
+    "identity": "${clientId(_ga)}",
+    "viewId": "PAGE_VIEW_ID_64"
   },
   "requests": {
-    "track": "https://${namespace}.amp.permutive.com/track?k=${key}&i=${identity}&it=amp",
+    "track": "https://${namespace}.amp.permutive.com/track?k=${key}&i=${identity}&it=amp&vid=${viewId}&s=${viewId}",
```

Comment:
Is it expected to send the `${viewId}` both under `vid` and `s`?

Reply:
Thanks for looking at this @zhouyx. Yes, for now we want to use `viewId` for both `vid` (which represents the concept of a single page view) and `s` (which we use to represent a session of a fixed amount of time, potentially encompassing multiple page views). The most similar thing I could find was this [client id](https://github.com/ampproject/amphtml/blob/master/spec/amp-var-substitutions.md#client-id), which doesn't really fit our use case, is there a better way of representing something like that?

Reply:
I see. You're right, unfortunately there's no session id support. This is because the AMP page has limited access to the client side storage when served from the cache. related to #18931